src/Npgsql/NpgsqlWriteBuffer.cs

```diff
@@ -531,7 +531,7 @@ void Throw()
                 if (_messageLength is null)
                 {
                     Connector.Break();
-                    new InvalidOperationException("No message was started");
+                    throw new InvalidOperationException("No message was started");
```

Comment:
Seems like this missing throw also exists in 4.1 https://github.com/npgsql/npgsql/blob/67acbe027e28477ac2199e15cfb554bb2ffaf169/src/Npgsql/NpgsqlWriteBuffer.cs#L562

Missed it during reviewing @roji.

Reply:
Oh oops :(

Reply:
It's really not a big deal for 4.1 since it just means we're not throwing a nice error if we missed `StartMessage` somewhere.

Reply:
No I agree, it's not worth another release. Just thought to mention it.